govcd/extension.go

```diff
@@ -33,28 +34,70 @@ func GetExternalNetworkByName(vcdClient *VCDClient, networkName string) (*types.
 	return &types.ExternalNetworkReference{}, nil
 }
 
-func getExternalNetworkHref(vcdClient *VCDClient) (string, error) {
-	extensions, err := getExtension(vcdClient)
+// If user specifies a valid external network name, then this returns a
```

Comment:
Good practice to start function comment with it's name to be go doc friendly. Otherwise it breaks grammar as in the example below.

<img width="1104" alt="image" src="https://user-images.githubusercontent.com/15804230/57283210-53470480-70b7-11e9-8f84-c4a3369008a2.png">

Reply:
fixed